fpptatweaks.php

```diff
@@ -477,3 +477,31 @@ function fpptatweaks_civicrm_postProcess($formName, $form) {
     }
   }
 }
+
+/**
+ * Implements hook_civicrm_searchColumns().
+ *
+ */
+function fpptatweaks_civicrm_searchColumns( $objectName, &$headers,  &$rows, &$selector ) {
+  // Check if it is seasrch contribution
+  if ($objectName == 'contribution') {
+    // Insert additional column header in the tab
+    $insertedHeader = [
+      'name' => E::ts('Inv #'),
+      'field_name' => 'invoice_number',
+      'direction' => CRM_Utils_Sort::DONTCARE,
+      'weight' => 45,
+    ];
+    $headers[7] = $insertedHeader;
```

Comment:
Be careful here. We don't know if some other extension may already have defined `$headers[7]`. I think you should be able to just use `$headers[]`, which will append a new item to the `$headers` array without needing to know the array index.